adwaita/css/widgets/dialogs.css

```diff
@@ -34,7 +34,7 @@ html.client_chat_frame
 		&:has(> .GenericConfirmDialog:not(._2TAQYpbxatDYN3Ex76KX5u):not(.q5OA-x6LcVkvtLCGp-8JC)),
 		&:has(.CreateChatChannelDialog)
 		{
-			border-radius: var(--adw-alert-dialog-radius) !important;
+			--adw-dialog-radius: var(--adw-alert-dialog-radius) !important;
```

Comment:
Is there a reason to override a var instead of setting `border-radius` directly and keeping `border-radius: inherit` for the outline?